sdk/dnsresolver/Azure.ResourceManager.DnsResolver/tests/DnsResolverTestBase.cs

```diff
@@ -8,13 +8,15 @@
 using Azure.ResourceManager.TestFramework;
 using Azure.ResourceManager.Network;
 using NUnit.Framework;
+using Azure.ResourceManager.Network.Models;
 
 namespace Azure.ResourceManager.DnsResolver.Tests
 {
     public class DnsResolverTestBase : ManagementRecordedTestBase<DnsResolverManagementTestEnvironment>
     {
         protected string SubnetName => "snet-endpoint";
-
+        public string DefaultVnetID;
+        public string DefaultSubnetID;
```

Comment:
Could we change the type of these variables to `ResourceIdentifier`?

Reply:
fixed